docs/conf.py

```diff
@@ -20,7 +22,7 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = ["_bleio", "adafruit_ble.services"]
```

Comment:
Please remove these mocks and the sys.path above. The real packages in requirements.txt should be enough.

The goal is to get this OK without mocks so that we don't need mocks copied into each BLE library separately.

Reply:
If the `_bleio` mock in `adafruit_ble` is enough, yes, that's good.

Reply:
@tannewt Ah, that makes a lot of sense. I totally forgot that actions will install the dependencies before it builds the docs